okhttp-logging-interceptor/src/main/java/okhttp3/logging/HttpLoggingInterceptor.kt

```diff
@@ -44,7 +43,8 @@ class HttpLoggingInterceptor @JvmOverloads constructor(
 
   @Volatile private var headersToRedact = emptySet<String>()
 
-  @Volatile @set:JvmName("-deprecated_setLevel") var level = Level.NONE
+  @set:JvmName("level")
```

Comment:
Is it good to define `@get:JvmName("level")` to match the setter name? 🤔WDYT? :)

Reply:
this is necessary to avoid a conflict with the `setLevel()` method that returns a `HttpLoggingInterceptor`.